jbpm-wb-executor-service/jbpm-wb-executor-service-client/pom.xml

```diff
@@ -48,6 +48,10 @@
       <artifactId>errai-common</artifactId>
     </dependency>
     <dependency>
+      <groupId>org.jboss.errai</groupId>
```

Comment:
@nmirasch can you double check if these new dependencies are needed? I could compile without it.